solutions/day-10/day10Bayu.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+    class Solution {
+
+        /**
+         * @param Integer[] $numbers
+         * @param Integer $target
+         * @return Integer[]
+         */
+        function twoSum2($numbers, $target): array|null {
+            $checked = [];
+
+            for($i = 1; $i <= count($numbers); $i++){
+                $currentValue = $numbers[$i -1];
+                $complement = $target - $currentValue;
+
+                if(isset($checked[$complement])){
+                    return [$checked[$complement], $i];
+                }
+
+                $checked[$currentValue] = $i;
+            }
+            return null;
+        }
```

Comment:
Ini bener mas, dan udah efektif sebenarnya. Tapi coba pake two pointer mas, since item nya udah keurut. Harusnya bisa lebih optimize. Tapi kalo mau pake cara ini juga gapapa sih, wdyt mas?